lib/Model/events.js

```diff
@@ -221,28 +248,59 @@ Model.prototype.removeContextListeners = function(value) {
   return this;
 };
 
-function eventListener(model, subpattern, cb) {
+/**
+ * @param {Model} model
+ * @param {string} eventType
+ * @param {string | Model} subpattern
+ * @param {Function} cb
+ * @param {ModelOnOptions} [options]
+ */
+function eventListener(model, eventType, subpattern, cb, options) {
+  options = options || {};
   if (cb) {
-    // For signatures:
-    // model.on('change', 'example.subpath', callback)
+    // For signatures with pattern:
+    // model.on('change', 'example.subpath.**', callback)
     // model.at('example').on('change', 'subpath', callback)
-    var pattern = model.path(subpattern);
-    return modelEventListener(pattern, cb, model._eventContext);
+    if (options.useEventObjects) {
+      var pattern = model.path(subpattern);
+      return modelEventListener(eventType, pattern, cb, model._eventContext);
+    } else { // eslint-disable-line no-else-return
```

Comment:
Rather than this eslint rule, you can just get rid of the `else` block, since the if above returns